specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-10-01/offers.json

```diff
@@ -0,0 +1,302 @@
+          },
+          {
+            "$ref": "#/parameters/ODataExpandParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Offer"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Offer": {
+            "$ref": "./examples/GetOffer.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OfferList": {
+      "description": "List of Offer proxy resources for the HCI cluster.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of Offer proxy resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Offer"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Offer": {
+      "description": "Offer details.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Offer properties.",
+          "$ref": "#/definitions/OfferProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "OfferProperties": {
+      "description": "Publisher properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning State",
+          "type": "string",
+          "readOnly": true
+        },
+        "publisherId": {
+          "type": "string",
+          "description": "Identifier of the Publisher for the offer"
+        },
+        "content": {
+          "type": "string",
+          "description": "JSON serialized catalog content of the offer"
+        },
+        "contentVersion": {
+          "type": "string",
+          "description": "The API version of the catalog service used to serve the catalog content"
+        },
+        "skuMappings": {
+          "type": "array",
+          "description": "Array of SKU mappings",
+          "items": {
+            "$ref": "#/definitions/SkuMappings"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "SkuMappings": {
+      "description": "SKU Mapping details.",
+      "type": "object",
+      "properties": {
+        "catalogPlanId": {
+          "type": "string",
+          "description": "Identifier of the CatalogPlan for the sku"
+        },
+        "marketplaceSkuId": {
```

Comment:
>marketplaceSkuId [](http://example.com/codeflow?start=9&length=16)

Same.

Reply:
Will not fix

Reply:
Please improve the description by disambiguating the Id term (e.g. Resource Id, Guid, Client selected name, etc.) before merging this PR.